src/deployment-guides/tko-on-vsphere-with-tanzu.md

```diff
@@ -110,200 +105,216 @@ Do the following to deploy NSX Advanced Load Balancer Controller node:
+   - Customize the configuration by providing Management Interface IP Address, Subnet Mask, and Default Gateway. The remaining fields are optional and can be left blank.
 
-   The following example shows the final configuration of the NSX Advanced Load Balancer Controller node.
+   The following example shows the final configuration of the NSX Advanced Load Balancer controller node.
 
-   ![](./img/tko-on-vsphere-with-tanzu/image37.png)
+   ![Completed configuration of the NSX Advanced Load Balancer Controller node](img/tko-on-vsphere-with-tanzu/TKO-VWT01.png)
 
  For more information, see the product documentation [Deploy the Controller](https://docs.vmware.com/en/VMware-vSphere/7.0/vmware-vsphere-with-tanzu/GUID-CBA041AB-DC1D-4EEC-8047-184F2CF2FE0F.html).
 
 ### Configure the Controller Node for your vSphere with Tanzu Environment
 
-After the Controller VM is deployed and powered-on, configure the Controller VM for your vSphere with Tanzu environment. The Controller requires several post-deployment configuration parameters.
+After the controller VM is deployed and powered-on, configure the controller VM for your vSphere with Tanzu environment. The controller requires several post-deployment configuration parameters.
 
-1. On a browser, go to https://<_controller node01-fqdn_>/.
+On a browser, go to https://<https://<alb-ctlr01.tanzu.lab>/.
 
 1. Configure an **Administrator Account** by setting up a password and optionally, an email address.
 
-  ![](./img/tko-on-vsphere-with-tanzu/image11.jpg)
+  ![Screenshot of Create Account screen](img/tko-on-vsphere-with-tanzu/TKO-VWT02.png)
 
-1. Configure **System Settings** by specifying the backup passphrase and DNS information.
+2. Configure **System Settings** by specifying the backup passphrase and DNS information.
 
-  ![](./img/tko-on-vsphere-with-tanzu/image3.jpg)
+  ![Screenshot of System Settings screen](img/tko-on-vsphere-with-tanzu/TKO-VWT03.png)
 
-1. (Optional) Configure **Email/SMTP**.
+3. (Optional) Configure **Email/SMTP**.
 
-  ![](./img/tko-on-vsphere-with-tanzu/image56.png)
+  ![Screenshot of Email/SMTP screen](img/tko-on-vsphere-with-tanzu/TKO-VWT04.png)
 
-1. Configure Multi-Tenant settings as follows:
+4. Configure **Multi-Tenant** settings as follows:
 
    - IP Route Domain: Share IP route domain across tenants.
    - Service Engine Context: Service Engines are managed within the tenant context, not shared across tenants.
 
-   ![](./img/tko-on-vsphere-with-tanzu/image4.jpg)
+   ![Screenshot of Multi-Tenant screen](img/tko-on-vsphere-with-tanzu/TKO-VWT05.png)
+
+5. Click on **Save** to finish the post-deployment configuration wizard.
+
+   If you did not select the **Setup Cloud After** option <_I don't see this option_> before saving, the initial configuration wizard exits. The Cloud configuration window does not automatically launch and you are directed to a Dashboard view on the controller.
```

Comment:
```suggestion
   If you did not select the **Setup Cloud After** option before saving, the initial configuration wizard ends. The cloud configuration window does not automatically launch and you are directed to a dashboard view on the controller.
```

Reply:
I removed my embedded comment **<_I don't see that option_>** from the MD file so I could explain it here, which is a better place for comments. When I reviewed this guide, I didn't see where the **Setup Cloud After** option is.  Was it shown in one of the screenshots you replaced, or did I just miss it? Please verify that the option is still in the wizard, and if it is, consider rewording this line so it's clearer.